xsensmt/CMakeLists.txt

```
@@ -26,4 +27,9 @@ if(ENABLE_xsensmt)
          
   yarp_install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/xsensmt.ini"  DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
 
+  # Install the example file from Xsens
+  add_executable(xsensExample exampleScanDevicesAndReceiveData.cpp)
+  target_link_libraries(xsensExample xsens_mt_software_suite)
+  install(TARGETS xsensExample RUNTIME DESTINATION bin)
```

```suggestion
  # Install the example file from Xsens
  add_executable(yarp-xsensmt-example exampleScanDevicesAndReceiveData.cpp)
  target_link_libraries(yarp-xsensmt-example xsens_mt_software_suite)
  install(TARGETS yarp-xsensmt-example RUNTIME DESTINATION bin)
```

If we want to keep the executable (I guess for  future debugging) it is better to use an executable with a non-ambigous name.

---

Good point, suggestion committed!